packages/kmod/kmod.spec

```diff
@@ -27,6 +27,27 @@ cp COPYING COPYING.LGPL
 cp tools/COPYING COPYING.GPL
 
 %build
+
+mkdir static-build
+pushd static-build
+
+%define _configure ../configure
```

Comment:
Nit: might be slightly less "magical" to always define configure like this, and to make a `dynamic-build` dir to hold the dynamically linked objects.

But I'm fine with this too.

Reply:
Fair enough. I'll update the steps to include a `dynamic-build` and make it more straightforward